static/preview-code.css

```diff
@@ -0,0 +1,8 @@
+@import url('https://cdnjs.cloudflare.com/ajax/libs/highlight.js/11.8.0/styles/atom-one-dark.min.css');
```

Comment:
Could we get this from jsdelivr as well? At the moment the CSP doesn't allow cdnjs.cloudflare.com